src/Maestro/Maestro.Contracts/IPullRequestActor.cs

```diff
@@ -11,5 +11,12 @@ namespace Maestro.Contracts;
 public interface IPullRequestActor : IActor
 {
     Task<string> RunActionAsync(string method, string arguments);
-    Task UpdateAssetsAsync(Guid subscriptionId, int buildId, string sourceRepo, string sourceSha, List<Asset> assets);
+
+    Task UpdateAssetsAsync(
+        Guid subscriptionId,
+        int buildId,
+        string sourceRepo,
+        string sourceSha,
+        List<Asset> assets,
+        bool isCodeFlow);
```

Comment:
Can we introduce an enum rather than using the bool flag? Or what about having two functions: one for codeflow and another one for ... normal PR?
We cannot force clients of this function to always specify `UpdateAssetsAsync(..., isCodeFlow: false)` and having the bool flag usually harms readability.

Reply:
I am too afraid to do 2 methods because this one is executed via this reflection framework across the service fabric cluster. So it's registered as this `ActionMethod` and invoked dynamically on the actor.

I agree on the readability so I will think of a valid name for an enum.

Reply:
I changed to an enum. Wanted to make it a flag but it's better as a regular enum because of the serialization and backward compatibility.